### PR TITLE
New and updated rules

### PR DESCRIPTION
**Pull request by KommX**

### CODE REVIEW

File: install-dev-dependencies.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@
 set -o errexit -o xtrace
 if type apt-get >/dev/null ; then
   sudo apt-get install libxml2-dev libxml2-utils libxslt1-dev python-dev \
-    firefox chromium-browser zip sqlite3 python-pip libcurl4-openssl-dev
+    firefox chromium-browser zip sqlite3 python-pip libcurl4-openssl-dev libssl-dev
```
Comment (reedy) [inline install-dev-dependencies.sh]: Out of interest, why do you need to add libssl-dev in?

@jsha I'm guessing you're ok with this change?

Comment (KommX) [inline install-dev-dependencies.sh]: On a Ubuntu system (that I was running in a VM) I had the problem that running install-dev-dependencies.sh ended in a compiler error not finding "crypto.h" (or something similar). Installing libssl-dev solved this problem for me.

But on another machine running Ubuntu I can't reproduce this right now?!

**Comment by KommX** [comment]
I will close this and open up separate pull request.

File: install-dev-dependencies.sh (hunk 1 above)
Comment (jsha) [inline install-dev-dependencies.sh]: The machine on which you can't reproduce - does that already have libssl-dev?

I'd prefer to not add the dependency unless we're sure it's needed.

Comment (KommX) [inline install-dev-dependencies.sh]: Now I can't reproduce anymore.
We should just let this one go.

**Pull request closed without merging**